src/remote_code_ptr.h

```diff
@@ -60,6 +60,19 @@ class remote_code_ptr {
         return *this;
     }
   }
+  remote_code_ptr continue_executed_bkpt(SupportedArch arch) {
```

Comment:
This I think needs a better name. I have no idea what it's supposed to do.

Reply:
I'm very bad at coming up with names so I'm very open to suggestions....

I was initially going to call it "redo_****" but that feels really like making it redo the breakpoint, which is exactly the opposite.... so...

Reply:
advance_past_executed_bkpt?

Reply:
that sounds good to me.

Also put the aarch64 case above default ...

Reply:
Sure,

> Also put the aarch64 case above default ...

What about the undo function? I've got confused by that order before and I just copied it here.....

Reply:
Yeah you can fix that too.

Reply:
Done. It seems that the ordering could be for the `RR_FALLTHROUGH`. I changed that to a unreachable instead as already used elsewhere.